### ISSUE TITLE
@Loggable for all sub classes

### ISSUE THREAD

**Issue by eduanb** [jcabi/jcabi-aspects#218]
I am working on a spring mvc project and all my controllers inherit from an abstract. I would like to add @Loggable to the parent and have it propagate down to each concrete controller.

**Comment by dmarkov** [comment]
@yegor256 please do something about this issue

**Comment by yegor256** [comment]
@eduanb good idea, but it's technically not possible with AspectJ, as far as I know...

**Comment by eduanb** [comment]
It is actually quite simple with the + syntax. Here is an example http://blog.espenberntsen.net/2010/03/20/aspectj-cheat-sheet/
